services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.base_user import AbstractBaseUser
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> None:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+
+
+def get_user(user_id: int) -> AbstractBaseUser:

```

Comment:
The `get` method will raise a `DoesNotExist` exception if the user is not found. Consider handling this exception to avoid runtime errors.